### PR TITLE
update camunda 8 to v8.6 and switch to spring-boot-starter-camunda-sdk

### PR DESCRIPTION
**Pull request by maxbehr801**

### CODE REVIEW

File: camunda-carbon-reductor-c8/pom.xml [hunk 1]
```diff
@@ -54,22 +58,6 @@
...
-        <dependency>
-            <groupId>jakarta.validation</groupId>
-            <artifactId>jakarta.validation-api</artifactId>
-            <version>${version.jakarta.validation-api}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.hibernate.validator</groupId>
-            <artifactId>hibernate-validator</artifactId>
-            <version>${version.hibernate-validator}</version>
-        </dependency>
```
Comment (CRoberto1926) [inline camunda-carbon-reductor-c8/pom.xml:72]: These are now transitive?

Maybe you can also delete some of these ${version.[...]} variables?

File: pom.xml [hunk 2]
```diff
@@ -41,8 +41,9 @@
         <version.okhttp>4.12.0</version.okhttp>
         <version.hibernate-validator>8.0.1.Final</version.hibernate-validator>
         <version.jakarta.validation-api>3.0.2</version.jakarta.validation-api>
-        <version.zeebe.spring-boot>8.5.0</version.zeebe.spring-boot>
-        <version.camunda.spring-boot>7.22.0</version.camunda.spring-boot>
+        <version.camunda8>8.6.3</version.camunda8>
+        <version.camunda8.connector>8.6.2</version.camunda8.connector>
```
[the remark this answers is not in the document]
Comment (maxbehr801) [reply pom.xml:45]: These are used in the c7 and c8 submodules.

File: camunda-carbon-reductor-c8/pom.xml (hunk 1 above)
Comment (maxbehr801) [reply camunda-carbon-reductor-c8/pom.xml:72]: The dependencies are leftover when the Carbon Reductor was a "proper" connector. As the dependencies are no longer in use it's time to finally get rid of them.

**Review by CRoberto1926 — APPROVED** [review]
LGTM

**Pull request merged**